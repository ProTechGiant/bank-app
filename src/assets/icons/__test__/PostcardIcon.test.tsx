import { create } from "react-test-renderer";

import { PostcardIcon } from "../PostcardIcon";

describe("icon/PostcardIcon", () => {
  it("renders a PostcardIcon", () => {
    const result = create(<PostcardIcon />);

    expect(result).toMatchSnapshot();
  });
});
