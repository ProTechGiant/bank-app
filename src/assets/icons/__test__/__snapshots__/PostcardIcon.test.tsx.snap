// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/PostcardIcon renders a PostcardIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="17"
  bbWidth="17"
  fill="none"
  focusable={false}
  height={17}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 17,
        "width": 17,
      },
    ]
  }
  vbHeight={17}
  vbWidth={17}
  width={17}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M12.0417 7.79166H12.75C12.9379 7.79166 13.118 7.71703 13.2509 7.5842C13.3837 7.45136 13.4583 7.27119 13.4583 7.08333V6.375C13.4583 6.18713 13.3837 6.00697 13.2509 5.87413C13.118 5.74129 12.9379 5.66666 12.75 5.66666H12.0417C11.8538 5.66666 11.6736 5.74129 11.5408 5.87413C11.408 6.00697 11.3333 6.18713 11.3333 6.375V7.08333C11.3333 7.27119 11.408 7.45136 11.5408 7.5842C11.6736 7.71703 11.8538 7.79166 12.0417 7.79166ZM4.25 8.5H7.79167C7.97953 8.5 8.1597 8.42537 8.29253 8.29253C8.42537 8.15969 8.5 7.97952 8.5 7.79166C8.5 7.6038 8.42537 7.42363 8.29253 7.29079C8.1597 7.15796 7.97953 7.08333 7.79167 7.08333H4.25C4.06214 7.08333 3.88197 7.15796 3.74913 7.29079C3.6163 7.42363 3.54167 7.6038 3.54167 7.79166C3.54167 7.97952 3.6163 8.15969 3.74913 8.29253C3.88197 8.42537 4.06214 8.5 4.25 8.5ZM15.5833 2.83333H1.41667C1.22881 2.83333 1.04864 2.90796 0.9158 3.04079C0.782962 3.17363 0.708334 3.3538 0.708334 3.54166V13.4583C0.708334 13.6462 0.782962 13.8264 0.9158 13.9592C1.04864 14.092 1.22881 14.1667 1.41667 14.1667H15.5833C15.7712 14.1667 15.9514 14.092 16.0842 13.9592C16.217 13.8264 16.2917 13.6462 16.2917 13.4583V3.54166C16.2917 3.3538 16.217 3.17363 16.0842 3.04079C15.9514 2.90796 15.7712 2.83333 15.5833 2.83333V2.83333ZM14.875 12.75H2.125V4.24999H14.875V12.75ZM4.25 11.3333H7.79167C7.97953 11.3333 8.1597 11.2587 8.29253 11.1259C8.42537 10.993 8.5 10.8129 8.5 10.625C8.5 10.4371 8.42537 10.257 8.29253 10.1241C8.1597 9.99129 7.97953 9.91666 7.79167 9.91666H4.25C4.06214 9.91666 3.88197 9.99129 3.74913 10.1241C3.6163 10.257 3.54167 10.4371 3.54167 10.625C3.54167 10.8129 3.6163 10.993 3.74913 11.1259C3.88197 11.2587 4.06214 11.3333 4.25 11.3333Z"
      fill={
        {
          "payload": 4280098077,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
